docs/architecture/microservices/microservice-ddd-cqrs-patterns/apply-simplified-microservice-cqrs-ddd-patterns.md

```diff
@@ -21,7 +21,7 @@ CQRS means having two objects for a read/write operation where in other contexts
 
 An example of this kind of service is the ordering microservice from the eShopOnContainers reference application. This service implements a microservice based on a simplified CQRS approach. It uses a single data source or database, but two logical models plus DDD patterns for the transactional domain, as shown in Figure 7-2.
 
-![The logical Ordering microservice includes its Ordering database, which can be or not in the same Docker host. Having the database in the same Docker host is good for development, but not for production.](./media/image2.png)
+![The logical Ordering microservice includes its Ordering database, which can be or not in the same Docker host. Having the database in the same Docker host is good for development, but not for production.](./media/apply-simplified-microservice-cqrs-ddd-patterns/simplified-cqrs-and-ddd-microservice.png)
```

Comment:
no and in file names
https://review.docs.microsoft.com/en-us/help/contribute/file-path-guidelines?branch=master#name-your-files

```suggestion
![The logical Ordering microservice includes its Ordering database, which can be or not in the same Docker host. Having the database in the same Docker host is good for development, but not for production.](./media/apply-simplified-microservice-cqrs-ddd-patterns/simplified-cqrs-ddd-microservice.png)
```